lib/std/Build/Step/Compile.zig

```diff
@@ -594,20 +594,20 @@ pub fn forceUndefinedSymbol(compile: *Compile, symbol_name: []const u8) void {
 
 /// Returns whether the library, executable, or object depends on a particular system library.
 /// Includes transitive dependencies.
-pub fn dependsOnSystemLibrary(compile: *const Compile, name: []const u8) bool {
+pub fn dependsOnSystemLibrary(compile: *Compile, name: []const u8) bool {
     var is_linking_libc = false;
     var is_linking_libcpp = false;
 
     var dep_it = compile.root_module.iterateDependencies(compile, true);
-    while (dep_it.next()) |module| {
-        for (module.link_objects.items) |link_object| {
+    while (dep_it.next()) |it| {
```

Comment:
`it` is usually reserved for iterators and not their produced values. Some other users of `iterateDependencies()` call it `item`, I'd go with `dep` like here:

https://github.com/ziglang/zig/blob/3929cac154d71a3e19fd028fc67c1d1d15823ca2/lib/std/Build/Step/Compile.zig#L1098-L1100

Reply:
https://github.com/ziglang/zig/pull/21309/commits/a2bb98f86d763c78ec3491dfa0b00cd2d7f64ef4